pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1073,6 +1077,19 @@ func getKafkaConfig(kafka *api.KafkaExtObject) {
 	}
 }
 
+func getKafkaJBPMConfig(kafkaExt *api.KafkaExtObject, kafka *api.KafkaJBPMObject) {
+	// The emitter is enabled only if the kafka extension is enabled
+	if kafkaExt != nil && kafka != nil {
+		//if something is missing we set mandatory defaults
+		if kafka.MaxBlockMs == nil {
+			kafka.MaxBlockMs = Pint32(2000)
+		}
+		if len(kafka.BootstrapServers) == 0 {
+			kafka.BootstrapServers = "localhost:9092"
+		}
```

Comment:
some of the KafkaJbpmEventEmitter properties are missing, if we are providing defaults, we should add there all of them (same as we have for Kafka extension)

Reply:
we are using default sonly on the properties we know the defaults, bootstraps and acks

Reply:
I don't like this default value for bootstrap servers, for ocp, it will never work.
better make the property mandatory and fail the reconcile loop if it is not set.